api/__init__.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, Iterator, Type, Tuple, List
+import pathlib
+import importlib
+import sys
+import logging
+
+import pydantic
+
+_logger = logging.getLogger(__name__)
+
+SCHEMAS_PATH = pathlib.Path(__file__).parent / "schemas"
+
+
+def load_public_schemas() -> List[Type[pydantic.BaseModel]]:
+    """Loads all pydantic Schemas for export.
+
+    If a class is prefixed with a '_' (i.e. _MySchema), it will not be exported
+    as a public schema.
+
+    Performs a bit of python magic to load all modules in the api/ folder and
+    find all subclasses of the base model.
+
+    Returns: List of api schema specifications.
+    """
+    # Importing all python files in api/ to collect all schemas for public api.
+    # This is needed to load all subclasses of pydantic.BaseModel.
+    # If a module is not in the global modules, cls.__subclasses__
+    # will not be able to locate it.
```

Comment:
I don't python well, but I think this comment could be clearer / more explicit, e.g.:

> Find all python files in api/ and load them as global modules. 
> This allows us to just enumerate pydantic.BaseModel.__subclasses__ to find all our model classes.